### ISSUE TITLE
Include fits version dependency in README

### ISSUE THREAD

**Issue by tampakis** [projecthydra-labs/hydra-works#205]
Currently 0.6.2

**Comment by grosscol** [comment]
Currently hydra-derivatives requires fits 0.6.2.  The characterization included here was built against output from fits.sh 0.8.4, but it worked reasonably well for 0.6.2 output.  The tests now check that they work for some 0.6.2 output.

**Comment by jcoyne** [comment]
How is hydra-derivatives requiring fits 0.6.2?

**Comment by grosscol** [comment]
Yes.  Require was probably the wrong word for that.  It states it as a dependency in the readme.
